### DISCUSSION TITLE
Is managed databases allowed?

### DISCUSSION THREAD

**Discussion by Meminseeker** [bounswe/bounswe2024group11#164] [General]
Hello, dear teaching staff,

In the last lecture, @uskudarli said we can use platforms such as AWS, Digital Ocean, etc., for the deployment.
It is clear that we use managed compute machines to host our APIs. However, for the databases, we have two main options: using a fully managed database or using a managed compute machine and installing the database ourselves.

Are we allowed to use both options?

**Comment by furkan7258** [comment]
You can use a managed database if it's free.

**Reply by Meminseeker** [reply]
By free, do you mean to include the free credits of the CSPs such as AWS or Digital Ocean? 

In other words, for example, Digital Ocean gives free credits for two months. Using these credits, we can use their services for free. Giving free credits is common among cloud service providers (CSP).

I want to be sure you also accept it. Otherwise, it will be tough to find a truly free service 😅

**Reply by furkan7258** [reply]
Yes, everything will be done with free _credits_ or _time_.
